docker/fullnode-x/README.md

```diff
@@ -0,0 +1,18 @@
+
+0. Build the containers locally to your desktop/laptop with:
+   - `docker compose build`
+      - The build process can take a while to complete.
+
+1. First start postgres so you can create the DB with the `diesel` command:
+	- `docker compose up postgres -d`
+	  * **Note** that postgres will store its db data in `./postgres/data`
+	- `psql -U postgres -p 5432 -h localhost -c 'create database sui_indexer_testnet'`
+	- run these in sui.git/crates/sui-indexer:
+      * `diesel setup --database-url=postgres://postgres:admin@localhost:5432/sui_indexer_testnet`
+
+2. Copy the fullnode.yaml and genesis.blob files for the network to use and put them in the `fullnode/config/` folder.
```

Comment:
I think it might be good to offer a link to a resource for this. @randall-Mysten is https://docs.sui.io/build/fullnode the best public resource for pointing people at fullnode.yaml and genesis.blob?

Reply:
Agreed, but we'll add them later.